tests/parental_controls_setup.pm

```diff
@@ -90,13 +93,17 @@ sub run {
 
     sleep(2);
 
-    assert_screen('control_center_users_new_user_parentally_restricted', 10);
+    assert_screen('malcontent_control_new_user_parentally_restricted', 10);
 
-    # Change the user’s login language so that their ratings board changes from
-    # 7+ to 7. The default language which we are changing from is unset (so,
-    # the IARC ratings system).
+    # Switch back to the control centre and change the user’s login language so
+    # that their ratings board changes from
+    # Everyone to 3+. The default language which we are changing from is unset
```

Comment:
Why would the language affect the app suitability setting? And why would it be Everyone or 3+ if it was just set to 7+?

Reply:
The age brackets are designed to mirror the user’s locale’s ratings scheme (PEGI, USK, IARC, etc.) and are all mapped to an underlying age scheme called [CSM](https://www.commonsensemedia.org/).

The idea is that the restrictions for a user should be chosen in that user’s locale. It’s a massive corner case to be testing setting those restrictions from an admin account which is in a different locale (you’d expect all users on the same system to use the same locale), but changing the new user’s locale means we avoid having to change the test user’s locale and create a whole load of new needles for running the remainder of the test in a different language.

The comment above about 7+ is outdated and I’ll change it. It should say ‘Everyone’, and should mention ESRB rather than IARC.